### PR TITLE
test(STONEO11Y-49): add yamllint

### PR DESCRIPTION
**Pull request by srivickynesh**
add .yamllint config and yaml-lint to CI.

### CODE REVIEW

File: .tekton/pull-request.yaml [hunk 1]
```diff
@@ -74,6 +74,25 @@ spec:
...
+            - name: workspace
+          steps:
+            - name: yaml-lint
+              image: registry.access.redhat.com/ubi9/python-39:latest
+              workingDir: $(workspaces.workspace.path)
+              script: |
+                #!/bin/bash -ex
+
+                python -m pip install yamllint
+                cd $(workspaces.workspace.path)
```
Comment (yftacherzog) [inline .tekton/pull-request.yaml:94]: Do we need this line if we already specify `workingDir`?

File: .tekton/pull-request.yaml [hunk 2]
```diff
@@ -74,6 +74,25 @@ spec:
...
+                yamllint .
```
Comment (yftacherzog) [inline .tekton/pull-request.yaml:95]: Will it check every yaml on every sub directory?

Comment (srivickynesh) [reply .tekton/pull-request.yaml:95]: yes

File: .tekton/pull-request.yaml (hunk 1 above)
Comment (srivickynesh) [reply .tekton/pull-request.yaml:94]: just checking here to see if that is the root path is correct

File: .tekton/pull-request.yaml (hunk 2 above)
Comment (yftacherzog) [reply .tekton/pull-request.yaml:94]: ok. The guidelines will be something to be added to the contributing guide.

Also, when it comes to Prometheus unit tests, I didn't find a way yet to break long lines, and they tend to be very long. We need to either find a way to do it, or find a way to exempt them specifically from line length tests.

Comment (srivickynesh) [reply .tekton/pull-request.yaml:94]: yeah. I will also add the rule files for yamllint

Comment (yftacherzog) [reply .tekton/pull-request.yaml:94]: cool

Comment (srivickynesh) [reply .tekton/pull-request.yaml:94]: Removed the long-line errors

File: .tekton/pull-request.yaml (hunk 1 above)
Comment (srivickynesh) [reply .tekton/pull-request.yaml]: The root path is correct and removed it.

**Comment by yftacherzog** [comment]
lgtm

File: .yamllint [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  quoted-strings: disable
+  trailing-spaces: enable
+  truthy:
+    level: warning
+
+  # accept both     key:
+  #                   - item
+  #
+  # and             key:
+  #                 - item
```
Comment (yftacherzog) [inline .yamllint:33]: Do we need this?

Comment (srivickynesh) [reply .yamllint]: No, we don't need it. will remove it.

**Review by yftacherzog — APPROVED** [review]

File: .yamllint [hunk 4]
```diff
@@ -0,0 +1,29 @@
+ignore: |
+  /vendor
```
Comment (amisstea) [inline .yamllint:2]: Not a big deal but why is this needed?

**Review by amisstea — APPROVED** [review]

File: .yamllint (hunk 4 above)
Comment (srivickynesh) [reply .yamllint]: Removed
